WebAPI/Controllers/CategoriesController.cs

```diff
@@ -15,18 +15,32 @@ public CategoriesController(ICategoryService categoryService)
         _categoryService = categoryService;
     }
 
+    /// <summary>
+    /// Gets all Categories.
+    /// </summary>
+    /// <returns>List of Categories</returns>
     [HttpGet]
     public async Task<ActionResult<List<CategoryDetailDto>>> GetAll() =>
                 await _categoryService.GetAllAsync();
 
-    [HttpGet("{id}:guid")]
+    /// <summary>
+    /// Gets a specific Image.
+    /// </summary>
+    /// <param name="id"></param>
+    /// <returns>ImagePath</returns>
```

Comment:
In scope of XML-description if you are using general description try to follow normal rules of writing messages. If you want to refer to some code, use `<see>` or `<c>` tags.
In general, I would advise to use normal description as in `<summary>` for `<return>`